### ISSUE TITLE
Convert events to JSON files

### ISSUE THREAD

**Issue by szabgab** [szabgab/codeandtalk.com#248]
As the first step for https://github.com/afilina/dev-community-data/issues/90

Convert the data/events/* files to JSON format that matches the fields used in the JSON file of https://github.com/afilina/dev-community-data/

At first create one JSON file per event.

Then evaluate and see if the one JSON file per even series might be better or not.

**Comment by rollandf** [comment]
How can I help with this issue ?

**Comment by szabgab** [comment]
Thanks for the offer, but I think I'll handle it myself in the next few days. Just keep adding more events :)

**Comment by ShaneCurcuru** [comment]
I agree having event sets as a concept in the data is important to some users.  My suggestion is to keep this, but make sure that it's as simple as possible for event submitters to keep their "most current" data updated, without having to understand the data structures. 

Half the battle is making it **super-simple** for FOSS community members from far-reaching events to self-report and also **self-update and maintain** the data format.

- Having the event set data provide automatic fallback values for per-event data is perfect; if the specific event fills it in, great; if not use the fallback.

- Any way that event organizers don't have to think about the event set data, especially when they're updating for new events, is important.  Lots of different users from different backgrounds and communities (hopefully) will be inputting data - so making it easy for all is big.

Thanks! This all sounds great.

**Comment by afilina** [comment]
@szabgab Do you mind if I write a script to automate the conversion? I'll then commit the resulting JSON files for each event series. Then it will be really easy to make the merge of the repos. I'm already familiar with the destination format and can adapt it based on additional variables you might have.

**Comment by szabgab** [comment]
@afilina be my guest 😃 . Will you also update the code that collects the data and generates the super-json file I use to serve the codeandtalk.com site?

**Comment by szabgab** [comment]
json conversion is done to per event files except for the twitter and hashtag fields that I raised here: https://github.com/afilina/dev-community-data/issues/90 

I'll convert them to per series json files later.
